Comp2022/src/main/cpp/subsystems/Drivetrain.cpp

```diff
@@ -710,8 +711,15 @@ void Drivetrain::MoveWithLimelightExecute(void)
     }
 
     // get turn value - just horizontal offset from target
-    double turnOutput = -m_turnPid.Calculate(robotContainer->m_vision.GetHorizOffsetDeg(), m_targetAngle);
-
+    double turnOutput = turnOutput = -m_turnPid.Calculate(robotContainer->m_vision.GetHorizOffsetDeg(), m_targetAngle);
+    if (tx > 0)

```

Comment:
I think we have the sign switched here - we want these values to move away from zero but as written I think we will be moving towards zero.

Let me give an example of what might happen now:
let's say tx is 1 (which is >0)
on line 714, turnOutput = -(tx * kp) = -(1 * 0.001ish) = -(0.001ish) = -0.001ish
On line 717 we add our turnConstant which might give us a positive result

Let's instead do it like this:
```
if (turnOutput > 0) {
   turnOutput = turnOutput + turnConstant
}
else if (turnOutput < 0) {
   turnOutput = turnOutput - turnConstant
}
```

Because we're acting on the resulting calculation, we don't even need to think about the sign here.